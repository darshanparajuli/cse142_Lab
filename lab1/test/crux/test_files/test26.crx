_gorn
__
GoRn
_0234_3
