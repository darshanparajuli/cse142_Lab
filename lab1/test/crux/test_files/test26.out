IDENTIFIER(_gorn)(lineNum:1, charPos:1)
IDENTIFIER(__)(lineNum:2, charPos:1)
IDENTIFIER(GoRn)(lineNum:3, charPos:1)
IDENTIFIER(_0234_3)(lineNum:4, charPos:1)
EOF(lineNum:5, charPos:1)
